pkg/remotewrite/validate.go

```diff
@@ -24,6 +24,10 @@ func FindClusterIDs(req *prometheus.WriteRequest) map[string]int {
 	return clusterID
 }
 
+func IsMetadataRequest(remoteWriteRequest *prometheus.WriteRequest) bool {
+	return remoteWriteRequest != nil && len(remoteWriteRequest.Timeseries) == 0 && len(remoteWriteRequest.Metadata) != 0
```

Comment:
`prometheus.WriteRequest` provides getters for Timeseries and Metadata. Could we use those instead?